src/main/java/gift/wishes/WishService.java

```diff
@@ -23,6 +26,19 @@ public List<Wish> findByMemberId(Long memberId){
         return wishRepository.findByMemberId(memberId);
     }
 
+    public Page<Wish> getAllWishPage(Long memberId){
+        Pageable pageable = PageRequest.of(0,10);
+        Page<Wish> allWishPage = wishRepository.findByMemberId(memberId,pageable);
+        return allWishPage;
+    }
+
+    public Page<Wish> getWishPage(Long memberId, int page){
+        Pageable pageable = PageRequest.of(page,10);
```

Comment:
IntelliJ 코드(라인) 자동 정렬 Command + Option + L 단축키 사용하시면 편리하게 자동적용 할 수 있습니다.
```suggestion
        Pageable pageable = PageRequest.of(page, 10);
```